lib/src/over_react_redux/over_react_redux.dart

```diff
@@ -497,14 +501,9 @@ class ReactJsReactReduxComponentFactoryProxy extends ReactJsContextComponentFact
 
 /// Converts a Redux.dart [Store] into a Javascript object formatted for consumption by react-redux.
 JsReactReduxStore _reduxifyStore(Store store) {
-  // Memoize this so that the same ReactInteropValue instances will be used
-  // for a given state, allowing JS `===` checks to not fail when the same
-  // state object is passed.
```

Comment:
Since we pass through the value directly (and memoize it in the cases we wrap it in `DartValueWrapper.wrapIfNeeded`), this is no longer necessary